src/lib.rs

```diff
@@ -220,12 +237,17 @@ fn locate_poi(
 
     poi.administrative_regions = admins;
     poi.address = poi_address;
-    poi.label = format_label(&poi.administrative_regions, &poi.name);
+    poi.label = format_poi_label(
+        &poi.name,
+        iter_admins(&poi.administrative_regions),
+        &country_codes,
+    );
     poi.labels = format_international_poi_label(
-        &poi.administrative_regions,
         &poi.names,
         &poi.name,
         &poi.label,
+        iter_admins(&poi.administrative_regions),
```

Comment:
I've seen this function being called a few times already. Just wondering why not just passing the slice directly?

Reply:
These `format_*` functions expect an iterator of `&mimir::Admin`, so I didn't find a better way :(

See https://github.com/CanalTP/mimirsbrunn/blob/master/src/labels.rs#L97

Reply:
Then let's say it's good to go.

Reply:
sorry that's because of me since in mimir sometimes we don't have a `Vec<Admin>` ([there](https://github.com/CanalTP/mimirsbrunn/blob/master/src/bin/bano2mimir.rs#L102-L106) for example)